sycl/include/sycl/stl_wrappers/__sycl_cmath_wrapper_impl.hpp

```diff
@@ -0,0 +1,541 @@
+//==------------- cmath-fallback.h -----------------------------------------==//
```

Comment:
Please, update the content (comments, macro) of the file to align with the new name.